.changeset/silly-bulldogs-complain.md

```diff
@@ -0,0 +1,24 @@
+---
+'@openfn/language-dhis2': minor
```

Comment:
this is a major changeset because we've introduced breaking changes (`post()` no longer exists!)

You can just replace `minor` with `major` in the file here